frontend/app/tests/unit/fixtures/history-event-counterparties.json

```diff
@@ -0,0 +1,220 @@
+{
+  "result": [
```

Comment:
Do you think we can reduce the content of the fixtures to the bare minimum that is required to test the functionality? This looks a bit like the full response from the API, we certainly don't need everything.